src/main/java/org/scijava/module/ModuleItem.java

```diff
@@ -205,4 +205,6 @@
 	/** Sets the item's current value with respect to the given module. */
 	void setValue(Module module, T value);
 
+	/** Gets the item's format*/
+    String getFormat();
```

Comment:
This new API will break all implementations of `ModuleItem` (except those extending `AbstractModuleItem`, of course). It requires increasing the major version of `scijava-common` to `3.x.x.`.